src/wings_hotkey.erl

```diff
@@ -128,7 +133,10 @@ event_handler(Ev0= #mousebutton{}, #cs{info=Str, st=St}) ->
         {yes,Xglobal,Yglobal,Mod} ->
             TweakBits = wings_msg:free_rmb_modifier(),
             case Mod band TweakBits =/= 0 of
-		true ->  wings_tweak:menu(Xglobal, Yglobal);
+		true ->  case sc of
```

Comment:
the atom sc will never match none, don't you get a warning about that or at least dialyzer will find bugs like that.